robottelo/ui/base.py

```diff
@@ -343,8 +379,8 @@ def wait_until_element(self, locator, timeout=12, poll_frequency=0.5):
             )
             return None
 
-    def wait_until_element_is_clickable(
-            self, locator, timeout=12, poll_frequency=0.5):
+    def wait_until_element_is_clickable(self, locator, timeout=12,
```

Comment:
Is this necessary? It seems to go against the way we have been formatting functions' arguments throughout the code.